Dockerfile.mysql

```diff
@@ -0,0 +1,14 @@
+# Official MySQL image
+FROM mysql:latest
+
+# ENVs for MySQL
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+# Port 3306 for MySQL
+EXPOSE 3306
+
+# Set the volume
+VOLUME /var/lib/mysql
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/1823d02c-0ac4-4bda-b787-c1638487da86)